editor/editor_help.cpp

```diff
@@ -915,13 +915,11 @@ void EditorHelp::_update_doc() {
 	// Properties overview
 	HashSet<String> skip_methods;
 
-	bool has_properties = cd.properties.size() != 0;
-	if (cd.is_script_doc) {
-		has_properties = false;
-		for (int i = 0; i < cd.properties.size(); i++) {
-			if (cd.properties[i].name.begins_with("_") && cd.properties[i].description.strip_edges().is_empty()) {
-				continue;
-			}
+	bool has_properties = false;
+	for (const DocData::PropertyDoc &prop : cd.properties) {
+		if (cd.is_script_doc && prop.name.begins_with("_") && prop.description.strip_edges().is_empty()) {
```

Comment:
I'd suggest combining this into `prop.overridden || ...` and then just removing the second if, don't think scripts can override properties anyway (the override here just deals with the default value as far as I know)

Reply:
I almost want to move `prop.name.begins_with("_") && prop.description.strip_edges().is_empty()` to some method on the `PropertyDoc` class to make this more readable. Like

`cd.is_script_doct && prop.is_undocumented_private` or something like that

Reply:
Is it used anywhere else? If not it's not really worth it I'd say

Reply:
doesn't look like it. I updated the code!

Reply:
The grouping is not needed, && has higher precedence